src/isIterable.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Checks whether the passed value is iterable.
+ *
+ * @func isIterable
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.18.0|v2.18.0}
+ * @category Type
+ * @sig * -> Iterable
+ * @param {*} val The value to test
+ * @return {boolean}
+ * @example
+ *
+ * RA.isIterable(['arrays', 'are', 'iterable']); //=> true
+ * RA.isIterable('strings are iterable, too');   //=> true
+ * RA.isIterable([]);                //=> true
```

Comment:
Pls don't use empty spaces to pad-end the resulting values.
Use following formatting from e.g. `isNan` function:
```js
RA.isNaN('NaN'); // => false
RA.isNaN(undefined); // => false
RA.isNaN({}); // => false
RA.isNaN('blabla'); // => false